detection/README.md

```diff
@@ -0,0 +1,41 @@
+# Object Detection
+
+This directory provides examples and best practices for building object detection systems. Our goal is to enable the users to bring their own datasets and train a high-accuracy model easily and quickly. To this end, we provide example notebooks with pre-set default parameters shown to work well on a variety of datasets, and extensive documentation of common pitfalls, best practices, etc.
+
+Object Detection is one of the main problems in Computer Vision. Traditionally, this required expert knowledge to identify and implement so called “features” that highlight the position of objects in the image. Starting in 2012 with the famous AlexNet paper, Deep Neural Networks are used to automatically find these features. This lead to a huge improvement in the field for a large range of problems.
+
+This repository uses [torchvision's](https://pytorch.org/docs/stable/torchvision/index.html) Faster R-CNN implementation which has been shown to work well on a wide variety of Computer Vision problems. See the [FAQ](FAQ.md) for an explanation of the underlying data science aspects.
+
+```diff
++ (August 2019) This is work-in-progress and more functionality and documentation will be added continuously.
+```
+
+
+## Frequently asked questions
+
+Answers to frequently asked questions such as "How does the technology work?" can be found in the [FAQ](FAQ.md) located in this folder. For generic questions such as "How many training examples do I need?" or "How to monitor GPU usage during training?" see the [FAQ.md](../classification/FAQ.md) in the classification folder.
+
+
+## Getting Started
```

Comment:
I dont think we need any further installation instructions. ive updated the environment.yml file in my branch to install torchvision. instructions on setting up the conda env and the jupyter kernels are already in the root level README.

Reply:
Perfect. Will update the text

Reply:
Do we want to have different conda envs for each use case scenario? If not, it would be great to put everything about setup at the root level, and use one env.

Reply:
Fully agree, at least at the moment we don't need different conda environments. Hence our setup instructions are actually already at the root level.